src/Sdk/DTPipelines/Pipelines/ObjectTemplating/PipelineTemplateEvaluator.cs

```diff
@@ -159,6 +159,32 @@ public class PipelineTemplateEvaluator
             return result;
         }
 
+        public List<ActionStep> LoadCompositeSteps(
```

Comment:
lets make sure to get these changes into the server at the same time so we dont forget and stomp over important changes later.

Reply:
Leaving open for visibility before merging.

Reply:
@ericsciple Wait, I should probably change `LoadCompositeSteps` to `LoadNestedSteps` since that is clearer?